app/src/main/java/com/example/ivor_hu/meizhi/widget/GirlsAdapter.java

```diff
@@ -26,8 +26,8 @@
 public class GirlsAdapter extends RecyclerView.Adapter<GirlsAdapter.MyViewHolder> {
     private static final String TAG = "GirlsAdapter";
 
-    private Context mContext;
-    private RealmResults<Image> mImages;
+    private final Context mContext;
+    private final RealmResults<Image> mImages;
```

Comment:
Hi there. Thank u for your pull request. I'm wondering have u tested about this part?